controllers/keystoneapi_controller.go

```diff
@@ -1195,8 +1195,9 @@ func (r *KeystoneAPIReconciler) generateServiceConfigMaps(
 			instance.Status.DatabaseHostname,
 			keystone.DatabaseName,
 		),
-		"enableSecureRBAC": instance.Spec.EnableSecureRBAC,
-		"ProcessNumber":    instance.Spec.HttpdCustomization.ProcessNumber,
+		"ProcessNumber":       instance.Spec.HttpdCustomization.ProcessNumber,
```

Comment:
not related to this PR, but @Deydra71 we might want to add some functional tests for the `ProcessNumber`?